packages/vx-axis/src/axis/Axis.js

```diff
@@ -4,38 +4,45 @@ import { Line } from '@vx/shape';
 import { Point } from '@vx/point';
 import { Group } from '@vx/group';
 import center from '../utils/center';
-import isLeft from '../utils/isLeft';
 import identity from '../utils/identity';
-import isHorizontal from '../utils/isHorizontal';
+import ORIENT from '../constants/orientation';
 
 export default function Axis({
   scale,
-  orient,
+  orientation,
   top = 0,
   left = 0,
   stroke = 'black',
   strokeWidth = 1,
   strokeDasharray,
-  fontSize = 10,
   numTicks = 10,
   tickFormat,
   tickStroke = 'black',
   tickLength = 8,
-  tickPadding = 2,
-  tickOffset = 0,
-  tickK = 1,
+  tickLabelOffset = 14,
   tickTransform,
-  tickTextAnchor = "start",
-  tickTextFontFamily = 'Arial',
-  tickTextFontSize = 10,
-  tickTextFill = 'black',
-  tickTextDx,
-  tickTextDy,
   hideAxisLine = false,
   hideTicks = false,
   hideZero = false,
+  labelComponent = (
+    <text
+      textAnchor="middle"
+      fontFamily="Arial"
+      fontSize={10}
+      fill="black"
+    >
+      default label
```

Comment:
let's nuke the `default label` text, it was a bad decision on my part to have a default label text. In the case where folks want to have no label they have to put `<AxisLeft label={''} />` which is super annoying and unnecessary.